transports/http/webhooks_handler_test.go

```diff
@@ -0,0 +1,111 @@
+package http
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"testing"
+
+	"github.com/libsv/bitcoin-hc/internal/tests/testpulse"
+	handler "github.com/libsv/bitcoin-hc/transports/http/handlers"
+)
+
+var webhookUrl = "http://localhost:8080/api/v1/webhook/notify"
+
+var preparedWebhook = handler.WebhookRequest{
+	Url: webhookUrl,
+	RequiredAuth: handler.RequiredAuth{
+		Type:  "BEARER",
+		Token: "test-token",
+	},
+}
+
+// TestCreateWebhookEndpoint tests the webhook registration.
+func TestCreateWebhookEndpoint(t *testing.T) {
+	//setup
+	pulse, cleanup := testpulse.NewTestPulse(t, testpulse.WithoutApiAuthorization())
```

Comment:
IMHO in tests, authorisation could be disabled by default.
And probably it is.
So this could be just:

```suggestion
	pulse, cleanup := testpulse.NewTestPulse(t)
```